gui/src/app/app-routing.module.ts

```diff
@@ -28,7 +29,12 @@ const routes: Routes = [
   {
     path: '**',
     redirectTo: '/notFound'
+  },
+  {
+    path: 'login',
```

Comment:
This path declaration has to be declared before the one for the `notFound` page, otherwise it won't work.